spec/unit/remote_system_spec.rb

```diff
@@ -138,5 +138,19 @@
         expect(remote_system.read_file("/foo")).to be_nil
       end
     end
+
+    describe "#inject_file" do
+      it "copies a file via scp" do
+        expect(Cheetah).to receive(:run).with("scp", "/usr/foobar", "root@remotehost:/tmp")
+        remote_system.inject_file("/usr/foobar", "/tmp")
+      end
+    end
+
+    describe "#remove_file" do
+      it "removes a file" do
+        expect(remote_system).to receive(:run_command).with("rm", "/tmp/foo", {:stdout=>:capture, :privileged=>true})
```

Comment:
Line is too long. [117/100]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.